src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -43,10 +44,10 @@ const ChannelImage = ({
       alignItems: 'center'
     },
     myPostNotificationImageContainer: {
-      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue
+      backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue
     },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `myPostNotificationImageContainer` and `anonPmNotificationImageContainer` has been changed. Ensure that these changes are reflected across the application for consistency and do not affect readability or accessibility.
```diff
- backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue
+ backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->